packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -187,4 +208,18 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
     )
 }
 
+/**
+ * Via the built-in Proxy object, modify the behaviour of each GET request for the given SCAPI clients
+ */
+const proxyGetRequests = (clients: ApiClients, handlers: ProxyHandler<any>) => {
```

Comment:
Currently I'm proxying only the GET requests. Should it be _all_ requests instead?